docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Deploy MemoryDB Cluster Using the ACK MemoryDB Controller"
+description: "Create managed MemoryDB Cluster from a Amazon Elastic Kubernetes Service (EKS) deployment."
+lead: "Create and use MemoryDB for Redis using Amazon Elastic Kubernetes Service (EKS)."
```

Comment:
nit: Please remove the trailing period, and update to be the same as the description. Thanks

Reply:
Updated the document.